src/middlewared/middlewared/etc_files/sysctl_config.py

```diff
@@ -0,0 +1,43 @@
+import os
+import subprocess
+
+
+def set_autotune_sysctl(middleware):
+    autotune_path = '/usr/local/bin/autotune'
+    if os.access(autotune_path, os.X_OK):
+        if middleware.call_sync('system.is_freenas'):
+            ret = subprocess.run(
+                [
+                    autotune_path, '-o', '--kernel-reserved=1073741824',
+                    '--userland-reserved=2417483648', '--conf', 'sysctl'
```

Comment:
the subprocess command is the same, only thing changing are integer values, perhaps these should be just a var and use subprocess outside the if?